### ISSUE TITLE
Arming security system results in "hub local alarms are currently suspended"

### ISSUE THREAD

**Issue by AndrewX192** [arcus-smart-home/arcusplatform#8]
When arming the security system via the arcus web ui, I receive the following error message:

"hub local alarms are currently suspended"

**Comment by AndrewX192** [comment]
I'm not sure if this is correct, but I was able to get the alarm functionality to work by inserting the following into the agent's database:

```
root@<2nd GEN HUB>:~# sqlite3 /data/iris/db/iris.db
sqlite> insert into config (key, value, lastUpdateTime, lastReportTime) values ('hubalarm:state','ACTIVE',1532298418657,1532298418657);
```

and then restarting the agent (killing the java process listed in ```ps```)

**Comment by msilvermanks** [comment]
Hi,

You can also use the Oculus tool (included in the tools directory) to directly "poke" various parameters including activating hub local alarms, although the way you did it works as well to get you moving forward :-)

Oculus connects to the client bridge URL in your local running Arcus platform environment (or a cloud environment once that is set up)

**Comment by AndrewX192** [comment]
I was able to get the Oculus tool to run, but I didn't see anything about alarms or hub state in there. I'll double check though. What's the expected path to enabling hub local alarms - is that supposed to get flipped once the user has configured a sensor + keypad, or some other set of conditions?

I had to manually put my account into the "COMPLETED" state to bypass the upsell prompt in the Iris web ui, so I'm unsure if that may have caused this problem
